tests/installation/grub_test.pm

```diff
@@ -0,0 +1,35 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use strict;
+use base "y2logsstep";
+use testapi;
+use bmwqemu ();
+
+sub run() {
+    up_and_down();
```

Comment:
the fix there is not stable enough. i test a solution for that, sometimes test were failing. better 1 min more to stabilize, then 1 build fail and hope that it will no fail